packages/terra-doc-template/examples/test-examples/TestComponent.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const propTypes = {
+  /**
+   * A string of text to be displayed first
+   */
+  text: PropTypes.string,
+  /**
+   * A string of text to be displayed second
+  */
+  otherText: PropTypes.string,
+};
+
+const defaultProps = {
+  text: 'Hi!',
+  otherText: 'Bye.',
+};
+
+const TestComponent = props =>
+  (<p>{props.text}, {props.otherText}</p>);
```

Comment:
I don't know that there is a strong standard for this, but you might consider:
```
const TestComponent = props => (
  <p>{props.text}, {props.otherText}</p>
);
```

I would choose whichever one you consider easier to read